collections/Fairfax/MS_Fairfax_16.xml

```diff
@@ -92,14 +109,662 @@
+                     <msItem>
+                        <locus>(fols. 199r–199*v)</locus>
+                        <author key="person_22158282">John Lydgate</author>
+                        <title>Beware of Doublenesse</title>
+                        <incipit>This worlde is ful of variaunce <lb/>In euery thing whoo taketh hede</incipit>
+                        <explicit>Sette on youre brest your self tassure <lb/>A myghty shelde of doublenesse</explicit>
+                        <finalRubric>Explicit</finalRubric>
+                        <bibl type="repertory"><ref target="https://www.dimev.net/record.php?recID=5793"><title>DIMEV</title> 5793</ref></bibl>
+                     </msItem>
+                     <msItem>
+                        <locus>(fols. 199*v–200v)</locus>
+                        <author key="person_22158282">John Lydgate</author>
+                        <title type="desc">A Prayer for King Henry VI and his Queen and the People</title>
+                        <rubric>Ab inimicis nostris defende nos criste</rubric>
+                        <incipit>Most souereigne lord o blessed crist ihesu <lb/>ffrom oure enemyes delyuer vs and oure foon</incipit>
+                        <explicit>And ay preserve vnder thy myghty honde <lb/>hym and hys moder thy peple and thy londe</explicit>
+                        <finalRubric>Explicit</finalRubric>
+                        <bibl type="repertory"><ref target="https://www.dimev.net/record.php?recID=3563"><title>DIMEV</title> 3563</ref></bibl>
+                     </msItem>
+                     <msItem>
+                        <locus>(fol. 201r)</locus>
+                        <title>Truth</title>
+                        <incipit>fflee fro the pres and dwelle with <sic>withfastnes</sic>
+                           <lb/>Suffise vn to thy good though hyt be smalle</incipit>
+                        <explicit>Weyve thy lust and let thy gooste the lede <lb/>And trouthe shall the delyuer hyt is no drede</explicit>
+                        <bibl type="repertory"><ref target="https://www.dimev.net/record.php?recID=1326"><title>DIMEV</title> 1326</ref></bibl>
+                     </msItem>
+                  </msContents>
+               </msPart>
+               <msPart>
+                  <msIdentifier>
+                     <altIdentifier type="partial">
+                        <idno type="part">MS. Fairfax 16, fols. 202–305</idno>
+                     </altIdentifier>
+                  </msIdentifier>
+                  <msContents>
+                     <textLang mainLang="en" otherLangs="la">Middle English and Latin</textLang>
+                     <msItem>
+                        <locus>(fols. 202r–300r)</locus>
+                        <author key="person_22158282">John Lydgate (attrib.)</author>
+                        <title>Reson and Sensuallyte</title>
+                        <note>Freely translated from <title>Les Echecs amoureux</title> (lines 1–4873)</note>
```

Comment:
This long text should probably have an authority record. Should some other items as well?

Reply:
It wouldn't be difficult to create IDs systematically for everything in DIMEV in a Bodleian MS – for another PR …